bot/exts/core/help.py

```diff
@@ -115,6 +119,9 @@ def __init__(
     def _get_query(self, query: str) -> Union[Command, Cog]:
         """Attempts to match the provided query with a valid command or cog."""
         command = self._bot.get_command(query)
+
+        log.info(query)
+        log.info(command)
```

Comment:
Were these just local debugs? They don't make a lot of sense without any explanation. I think it is best if we just remove them?